src/channel_server/management_functions.cpp

```diff
@@ -873,22 +869,22 @@ auto management_functions::packet(ref_ptr<player> player, const game_chat &args)
 					packet.add<int64_t>(value);
 				}
 				else if (character == 'i') {
-					if (value >= INT32_MAX || value <= INT32_MIN) {
-						chat_handler_functions::show_info(player, "Number '" + value_string + "' is not between INT32_MIN and INT32_MAX");
+					if (value < 0 || value >= UINT32_MAX) {
+						chat_handler_functions::show_info(player, "Number '" + value_string + "' is not between 0 and " + lexical_cast<string>(UINT32_MAX));
 						return chat_result::handled_display;
 					}
 					packet.add<int32_t>(static_cast<int32_t>(value));
 				}
 				else if (character == 's') {
-					if (value >= INT16_MAX || value <= INT16_MIN) {
-						chat_handler_functions::show_info(player, "Number '" + value_string + "' is not between INT16_MIN and INT16_MAX");
+					if (value < 0 || value >= UINT16_MAX) {
+						chat_handler_functions::show_info(player, "Number '" + value_string + "' is not between 0 and " + lexical_cast<string>(UINT16_MAX));
 						return chat_result::handled_display;
 					}
 					packet.add<int16_t>(static_cast<int16_t>(value));
```

Comment:
Is intentional that you don't cast to `uint16_t` here? Same for these other casts.